### PR TITLE
added pie chart component

### PR DESCRIPTION
**Pull request by aborenko1996**

### CODE REVIEW

File: frontend/src/components/common/PieChart.vue [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                chartOptions: {
+                    title: this.title,
+                    pieHole: this.pieHole
+                }
+            }
+        }
+    }
+</script>
+
+<style lang="scss" scoped>
```
Comment (TatsuUkraine) [inline frontend/src/components/common/PieChart.vue]: don't keep empty tags in SFC

File: frontend/src/components/common/PieChart.vue [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            GChart
+        },
+        props: {
+            title: {
+                type: String,
+                default: 'PieChart'
+            },
+            data: {
+                type: Array,
+                default: () => []
```
Comment (TatsuUkraine) [inline frontend/src/components/common/PieChart.vue]: make this prop required

Comment (StasKozar) [reply frontend/src/components/common/PieChart.vue]: Fixed

File: frontend/src/components/common/PieChart.vue (hunk 1 above)
Comment (StasKozar) [reply frontend/src/components/common/PieChart.vue]: Fixed

File: frontend/src/components/common/PieChart.vue (hunk 2 above)
Comment (TatsuUkraine) [reply frontend/src/components/common/PieChart.vue]: when prop is required, `default` is redundant

File: frontend/src/components/common/PieChart.vue [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+            },
+            pieHole: {
+                type: Number,
+                default: 0.95
+            },
+            legend: {
+                type: Object,
+                required: true,
+                default: () => [],
```
Comment (TatsuUkraine) [inline frontend/src/components/common/PieChart.vue]: same as prev required prop, default is redundant, also next time take in account that `[]` is not quite same wuth type `Object`

File: frontend/src/pages/Visitors.vue [hunk 4]
```diff
@@ -118,6 +150,10 @@
                 };
                 this.data.push(item);
             }
+            this.pieData.push(
```
Comment (TatsuUkraine) [inline frontend/src/pages/Visitors.vue]: not quite sure why you need updating pieData if you can define this in `data`

Comment (StasKozar) [reply frontend/src/pages/Visitors.vue]: Fixed

File: frontend/src/components/common/PieChart.vue (hunk 3 above)
Comment (StasKozar) [reply frontend/src/components/common/PieChart.vue]: Fixed

File: frontend/src/components/common/PieChart.vue (hunk 2 above)
Comment (StasKozar) [reply frontend/src/components/common/PieChart.vue]: Fixed

**Pull request merged**